src/types/typesAPI.ts

```diff
@@ -0,0 +1,38 @@
+import {SortingOrder, SortingType} from "../constantsAPI/constantsAPI";
+
+export interface ICar {
```

Comment:
interface is not a type. *.interfaces.ts, *.types.ts, *.enums.ts, *.constants.ts, etc.